apps/wpcom-block-editor/src/common/override-close-btn.js

```diff
@@ -0,0 +1,27 @@
+/* global calypsoifyGutenberg */
+
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { get } from 'lodash';
+
+domReady( () => {
+	const closeButtonInception = setInterval( function() {
+		const closeButton = document.querySelector( '.edit-post-fullscreen-mode-close__toolbar a' );
+
+		if ( ! closeButton ) {
+			return;
+		}
+
+		clearInterval( closeButtonInception );
+
+		// Add 'Checklist' label when the editor close button navigates back to checklist
+		if ( get( calypsoifyGutenberg, 'closeUrl', '' ).includes( '/checklist/' ) ) {
+			const checklistLabel = document.createElement( 'span' );
+			checklistLabel.className = 'checklist-label';
+			checklistLabel.innerHTML = 'Checklist';
```

Comment:
i18n? Maybe drop this part from the PR for now

Reply:
🤔 We probably need to update this value the server as part of the global script enqueue in wpcom. For example https://github.com/Automattic/wp-calypso/blob/20265560d69e24c9def3e1aa9995866dc38fef5a/apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php#L180

Though we probably don't need to tie this to FSE